### PR TITLE
Add 'meltmarshmallows' story

### PR DESCRIPTION
**Pull request by jaytiet**

### CODE REVIEW

**Comment by pmallory** [comment]
@casey-collab can you review this please?

File: english/meltmarshmallows/meltmarshmallows.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+You conjure up a flamethrower, and the sadistic side of you comes out.
```
Comment (casey-collab) [inline english/meltmarshmallows/meltmarshmallows.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Unfortunately, this pull request conflicts with another change. Could you merge the master branch of the original repository into your branch, resolve the merge conflicts, and update the pull request?

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: english/meltmarshmallows/meltmarshmallows.md (hunk 1 above)
Comment (jaytiet) [inline english/meltmarshmallows/meltmarshmallows.md]: Changed the page to be reachable

**Comment by jaytiet** [comment]
@casey-collab, can you check this?

**Comment by casey-collab** [comment]
Looks great! :shell:

**Pull request merged**